### PR TITLE
fixed MainqObj destruction

### PR DESCRIPTION
**Pull request by jvymazal**
not a solution but connected to #1491, and is nice to fix

### CODE REVIEW

**Comment by rgerhards** [comment]
This is just fixing the indention, right? Or am overlooking something?

**Comment by jvymazal** [comment]
Actually bit more than indentation, if you look closely on that diff you will see that the DestroyObj() and NULL assignment lines are now _inside_ the if(NULL) block, previously the were executed always. I did not do some extensive checking as to what problems could it cause, however according to both logic and present comment (in code) I would suggest we fix it.

File: runtime/glbl.c [hunk 1]
```diff
@@ -1021,9 +1021,9 @@ glblDestructMainqCnfObj(void)
 	/* Only destruct if not NULL! */
 	if (mainqCnfObj != NULL) {
 		nvlstChkUnused(mainqCnfObj->nvlst);
-	}
-	cnfobjDestruct(mainqCnfObj);
-	mainqCnfObj = NULL;
+        cnfobjDestruct(mainqCnfObj);
```
Comment (rgerhards) [inline runtime/glbl.c]: please fix indention (note that we use tabs for indention)

**Comment by rgerhards** [comment]
Ah, you are right, I overlooked this. Sorry for that. And, yes, I think this will make things work like intended.

**Comment by rgerhards** [comment]
Can you please squash the second commit (it's just a fix of the first) and update the first commit message to be a bit more up to the point. I tried to do this, but I do not have permissions to push my result back to the PR. Thanks.

**Pull request merged**